_data/roles.yml

```diff
@@ -77,6 +77,25 @@
       orcid: 0000-0002-7356-1779
       status: support
       start: "0000-00-00"
+- name: OBO Newsletter Steward
+  open: true
+  commitment: 2-4 hours per month
+  requirements:
+    - Should have a basic journalistic instinct, compiling the most relevant discussion points, decisions and issues for the wider community
+  description: You will be responsible for managing, compiling and disseminating the OBO Newletter
+  responsibilities:
+    - Manage a Google doc with the current and previous newsletter drafts based on the newsletter skeleton
+    - Adding GitHub issues and/or pull requests tagged with "newsletter" to the newsletter, with one or two sentences of summary
```

Comment:
I would make these command form ("Add", "Summarise", etc.) to match "Manage" and "send", but this is a minor thing and not critical.